graph/patterns/change-tracking.md

```diff
@@ -0,0 +1,94 @@
+# Change Tracking
```

Comment:
I'd prefer we drop the term change tracking and call this delta from the get-go.  change tracking seems like an unnecessary odata-ism to use for the broad feature area.

Reply:
we call it change tracking in our public docs and training, I thought aligning on that was good in case they wanted to do more research and get more information.

Reply:
I also think that Change Tracking is more generic and should stay  for consistency.

Reply:
great, resolving